keras/keras_integration.py

```diff
@@ -60,7 +62,7 @@ def create_model(trial):
     lr = trial.suggest_float("lr", 1e-5, 1e-1, log=True)
     model.compile(
         loss="categorical_crossentropy",
-        optimizer=keras.optimizers.RMSprop(lr=lr),
+        optimizer=RMSprop(lr=lr),
```

Comment:
Note that `lr` is deprecated; `learning_rate` is supposed to be appropriate.

Reply:
Do you want to follow that up in a separate PR? I'd happily review if you're okay with that.

Reply:
Thank you for kind pinging me. Sure, I'll send a follow-up PR.